pipeline/inputs/statsd.md

```diff
@@ -16,6 +16,13 @@ The plugin supports the following configuration parameters:
 | Listen | Listener network interface. | 0.0.0.0 |
 | Port | UDP port where listening for connections | 8125 |
 | Threaded | Indicates whether to run this input in its own [thread](../../administration/multithreading.md#inputs). | `false` |
+| Metrics | Ingested record will be marked as a metric record rather than a log record. | `off` |
+
+Note: When enabling `Metrics On`, we will also handle metrics from the DogStatsD protocol and the internal record in Fluent Bit will be handled as a metric type for downstream processing.
+[The full format of DogStatsD of metrics](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#the-dogstatsd-protocol) is not supported.
+Including key-value format of tags as below is supported:
+`<METRIC_NAME>:<VALUE>|<TYPE>|@<SAMPLE_RATE>|#<TAG_KEY_1>:<TAG_VALUE_1>`
```

Comment:
Can we add some example output as well showing what this would look like to stdout?

Reply:
Added in https://github.com/fluent/fluent-bit-docs/pull/1542/commits/20d10609d9210d05553ee2571dbd988c5a3829e3.